taprpc/assetwalletrpc/assetwallet.proto

```diff
@@ -398,6 +398,10 @@ message ProveAssetOwnershipRequest {
     bytes script_key = 2;
 
     taprpc.OutPoint outpoint = 3;
+
+    // An optional 32-byte challenge that may be used to bind the generated
+    // proof.
```

Comment:
Should we add something to the effect of "This challenge also needs to be presented when verifying the ownership proof, so it should be passed to the verifier, similar to how the message is needed when verifying a Bitcoin signature."?